charts/orb/Chart.yaml

```diff
@@ -10,7 +10,7 @@ name: orb
 description: Orb Observability Platform
 icon: https://avatars1.githubusercontent.com/u/13207490
 type: application
-version: 1.0.13
+version: 1.0.14
 appVersion: "0.9.0"
```

Comment:
let's sync `appVersion` with latest from https://github.com/ns1labs/orb/releases so 0.13.0. Actually we may want 0.14.0 this time because that is the current development version that necessitates the new redis.

Reply:
I bumped this to 0.14.0, and I will bump o orb repo to 0.14.0 too